app/models/address.rb

```diff
@@ -2,6 +2,16 @@ class Address < ApplicationRecord
   belongs_to :addressable, polymorphic: true, optional: true
 
   validates :city, :country, :state, :street, presence: true
+
+  def address
+    [street, city, state, country].compact.join(', ')
```

Comment:
Can one of them be nil?

Reply:
https://github.com/IF-115-Ruby/OSBB/blob/754aed5969c452f031c21daf086f1297baf1493d/app/models/address.rb#L4